backend/apps/search/tasks.py

```
@@ -62,9 +83,21 @@ def remove_model_from_search(app_label, model_name, object_id):
+        docs = SearchDocument.objects.filter(
             content_type=content_type, object_id=object_id
-        ).delete()
+        )
+        
+        for doc in docs:
+            # Delete from Meilisearch first
+            try:
+                index = get_meili_index()
+                if index:
+                    index.delete_document(str(doc.id))
+            except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)